README.md

```diff
@@ -655,38 +746,77 @@
     });
     ```
 
-  - Place 1 space before anonymous function parenthesis.
+  - Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
 
     ```javascript
-	// bad
-	function() {
-		var name;
-	}
+    // bad
+    if(isJedi) {
+      fight ();
+    }
+
+    // good
+    if (isJedi) {
+      fight();
+    }
+
+    // bad
+    function fight () {
+      console.log ('Swooosh!');
+    }
 
-	// good
-	function () {
-		var name;
-	}
+    // good
+    function fight() {
+      console.log('Swooosh!');
+    }
     ```
 
-  - Place an empty newline at the end of the file.
+  - Place 1 space before anonymous function parenthesis.
```

Comment:
@cmcculloh added this back in here.  Also, corrected anonymous functions throughout the readme.